lib/l10n/app_en.arb

```diff
@@ -951,5 +954,291 @@
+  "@connectionInterrupted": {
+    "description": "Message shown when the connection is interrupted during a download."
+  },
+  "connectionInterruptedBackground": "Connection was interrupted while downloading in the background.  This can be caused by OS settings.",
+  "@connectionInterruptedBackground": {
+    "description": "Message shown when the connection is interrupted during a download while in the background.  It advises checking OS settings, as they may cause background downloads to be blocked."
+  },
+  "connectionInterruptedBackgroundAndroid": "Connection was interrupted while downloading in the background.  This can be caused by enabling 'Enter Low-Priority State on Pause' or OS settings.",
+  "@connectionInterruptedBackgroundAndroid": {
+    "description": "Message shown when the connection is interrupted during a download while in the background on android.  It advises checking OS settings, as they may cause background downloads to be blocked, or checking the  'Enter Low-Priority State on Pause' setting."
+  },
+  "activeDownloadSize": "Downloading...",
+  "@activeDownloadSize": {
+    "description": "Message shown for download size while item is still being downloaded"
+  },
+  "missingDownloadSize": "Deleting...",
+  "@missingDownloadSize": {
+    "description": "Message shown for download size while item is still being downloaded"
+  },
+  "syncingDownloadSize": "Syncing...",
+  "@syncingDownloadSize": {
+    "description": "Message shown for download size while item is still syncing."
+  },
+  "runRepairWarning": "The server could not be contacted to finalize downloads migration.  Please run 'Repair Downloads' from the downloads screen as soon as you are back online.",
+  "@runRepairWarning": {
+    "description": "Message shown when an error occurs during the downloads repair step of the Isar downloads migration, which requires a server connection."
+  },
+  "downloadSettings": "Download Settings",
+  "@downloadSettings": {},
+  "showPlaylistSongs": "Show Items with Unknown Library.",
+  "@showPlaylistSongs": {
+    "description": "Title of setting that shows songs/albums from an unknown library in all libraries when on, or in no libraries when off."
+  },
+  "showPlaylistSongsSubtitle": "Some items may be downloaded with an unknown library.  They can show in all libraries, or none.",
+  "@showPlaylistSongsSubtitle": {
+    "description": "Description of setting that shows songs/albums from an unknown library in all libraries when on, or in no libraries when off."
+  },
+  "maxConcurrentDownloads": "Max Concurrent Downloads",
+  "@maxConcurrentDownloads": {
+    "description": "Title of setting controlling max concurrent downloads."
+  },
+  "maxConcurrentDownloadsSubtitle": "Increasing concurrent downloads may allow increased downloading in the background but will delay the effect of some settings changes and may cause some downloads to fail if very large, or cause lag in some cases.",
```

Comment:
The part about the delayed effect of settings changes isn't true anymore, is it?

Reply:
I believe it only meant requiresWifi, so yeah.  Removed.